// vitest/dist/browser.js
export { c as createExpect, d as describe, e as expect, i as it, s as suite, t as test } from './chunk-runtime-chain.e655f6cc.js';
import { e as dist } from './chunk-utils-import.ca62c9d7.js';
export { a as afterAll, d as afterEach, f as assertType, b as beforeAll, c as beforeEach, o as onTestFailed } from './chunk-utils-import.ca62c9d7.js';
export { a as setupGlobalEnv, s as startTests } from './chunk-runtime-setup.4c1b529e.js';
import * as chai from 'chai';
export { chai };
export { assert, should } from 'chai';
import 'util';
import './chunk-typecheck-constants.3f865d14.js';
import 'path';
import './chunk-utils-env.03f840f2.js';
import 'tty';
import 'url';
import 'local-pkg';
import './vendor-_commonjsHelpers.addc3445.js';
import './chunk-runtime-rpc.7959fc79.js';
import './chunk-utils-timers.793fd179.js';
import 'fs';
import './chunk-utils-source-map.5bbb50cd.js';
import './spy.js';
import 'tinyspy';
import 'perf_hooks';
import './chunk-integrations-coverage.befed097.js';
import './chunk-runtime-error.dfbbf9be.js';
import 'vite-node/source-map';
import './chunk-env-node.67948209.js';
import 'console';



var expectTypeOf = dist.expectTypeOf;
export { expectTypeOf };
